### ISSUE TITLE
how to debug it 

### ISSUE THREAD

**Issue by suchmors** [askldjd/iocp-server#1]
hello
I`ve downloaded your code,and release it.
it works very well
And I wanna combine my project with those codes,however when I choose debug x64 for it in vs2019,it gets failed,and said like this: 
error LNK1104	unable to find “libboost_thread-vc142-mt-sgd-x64-1_77.lib”  etc.

**Comment by askldjd** [comment]
It has been a while since I look into this library. If memory serves correctly, it statically link against Boost 1.42 (multithread). However, you can change it to link against different version.

Which version of boost do you have locally?

**Comment by suchmors** [comment]
yes I change to use boost 1.77 and i`ve sloved this problem now.
but the mean reason is not the version of boost, it is in the code.
first,the key code in Export.h:

//#if defined(_WIN32)
//#	if !defined(_DLL) || !defined(_MT)
//#		error This library must be used with the multi-threaded DLL CRT
//#	endif
//#endif
i just note them

and in ExternalLibraries.h：
//#define _HAS_ITERATOR_DEBUGGING 0

I change _HAS_ITERATOR_DEBUGGING  into 1 like that
#define _HAS_ITERATOR_DEBUGGING 1

then i can debug the code

very thanks to you,your code is so excellent!!!

**Comment by askldjd** [comment]
Interesting. I remember `_HAS_ITERATOR_DEBUGGING` was a big deal because it really slows down performance back in VS 2008. I am not sure if that's still the case.

I am glad that the library works for you. I wrote it couple jobs ago and the library went to production. If I recalled correctly, it was stress tested under a high throughput application and hardened pretty well under Windows 7.
